### ISSUE TITLE
Import fails for records with user id = 0

### ISSUE THREAD

**Issue by mkroon1** [LOVDnl/LOVD3#52]
Performing a file import with records owned by user id 0 results in an error.

To replicate this issue:
* Go to LOVD3 > Setup > Import data
* Select a text file with code below as content
* Select "Update existing data" as import mode
* Check "simulate"
* Click "import file"

~~~
### LOVD-version 3000-060 ### Full data download ### To import, do not remove or alter this header ###

## Variants_On_Genome ## Do not remove or alter this header ##
{{id}}	{{owned_by}}	{{VariantOnGenome/VIP}}
0000000015	00000	0
~~~

LOVD returns with the message:
> Error (Variants_On_Genome, line 5): Please select a valid entry from the 'owned_by' selection box, '00000' is not a valid value. Please choose from these options: ...

**Comment by ifokkema** [comment]
This is an example of the culprit:
https://github.com/LOVDnl/LOVD3/blob/master/src/class/object_genome_variants.php#L292
Sadly, each object that has an owner repeats this code...

**Comment by mkroon1** [comment]
I have a fix in d010334

This solves the current issue. The example file above does still not import properly however, it exits with the following message:

> No entries found that can be updated via the import file.

I guess this is a separate issue.
